kuma/javascript/src/breadcrumbs.jsx

```diff
@@ -36,20 +36,27 @@ export default function Breadcrumbs({ document }: DocumentProps) {
                 vocab="https://schema.org/"
                 aria-label="breadcrumbs"
             >
-                {document.parents.map((p, i) => (
+                {document.parents.map((p, i, { length }) => (
                     <li
                         key={p.url}
                         property="itemListElement"
                         typeof="ListItem"
                     >
                         <a
                             href={p.url}
-                            className="breadcrumb-chevron"
+                            className={
+                                i === length - 1
+                                    ? 'breadcrumb-previous'
+                                    : 'breadcrumb-chevron'
+                            }
                             property="item"
                             typeof="WebPage"
                             onClick={sendBreadcrumbItemClick}
                             onContextMenu={sendBreadcrumbItemClick}
                         >
+                            <span className="pre-text">
```

Comment:
I was concerned about disrupting the breadcrumb schema, so I wanted to leave `<span property="name">` as is.